source/_integrations/generic_thermostat.markdown

```diff
@@ -55,7 +55,7 @@ ac_mode:
 min_cycle_duration:
   description: Set a minimum amount of time that the switch specified in the *heater* option must be in its current state prior to being switched either off or on.
   required: false
-  type: [time, integer]
```

Comment:
This change is not correct, it accepts both time or an integer (which will be interpreted as seconds).

Reply:
Well, I explained that in the notes below. Can revert it back, but it looks ugly as time is not defined and that's the maing reason of all this fuss ;)